### PR TITLE
BZ-1771578: Docs for installing CLI using RPM

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1771578

Preview: https://bz-1771578--ocpdocs.netlify.com/openshift-enterprise/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands

### CODE REVIEW

File: modules/cli-installing-cli-rpm.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+[id="cli-installing-cli-rpm_{context}"]
+= Installing the CLI by using an RPM
+
+For Red Hat Enterprise Linux (RHEL), you can install the OpenShift CLI (`oc`) as an RPM if you have an active OpenShift Container Platform subscription on your Red Hat account.
```
Comment (bobfuru) [inline modules/cli-installing-cli-rpm.adoc]: s/OpenShift Container Platform/{product-title}/

File: modules/cli-installing-cli-rpm.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
```
Comment (bobfuru) [inline modules/cli-installing-cli-rpm.adoc]: If you are showing them using `oc` as root user, do you need to indicate this as a prereq?

File: modules/cli-installing-cli-rpm.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+----
+
+. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*OpenShift*'
+----
+
+. In the output for the previous command, find the pool ID for an
+{product-title} subscription and attach it:
```
Comment (bobfuru) [inline modules/cli-installing-cli-rpm.adoc]: "Attach it" to what?

**Review by bobfuru — COMMENTED** [review]
Few minor comments but otherwise LGTM!

File: modules/cli-installing-cli-rpm.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/cli-installing-cli-rpm.adoc]: Oops, copy paste error, nice catch!

File: modules/cli-installing-cli-rpm.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/cli-installing-cli-rpm.adoc]: Good suggestion

**Comment by zhouying7780** [comment]
LGTM , thanks.

**Pull request merged**